src/hotspot/share/cds/cdsConfig.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Incorrect copyright update - should be "2023, 2024,"

Reply:
Fixed.